pluto_build/05_qaqc.sh

```diff
@@ -3,13 +3,20 @@ source bin/config.sh
 
 # import previous version of pluto
 import_public dcp_pluto $VERSION_PREV 
```

Comment:
okay just writing this down so I can confirm with @SashaWeinstein. Since this comes from `.env` file, I don't have it for reference. `$VERSION_PREV` I am working off assumption this is 21v4

Reply:
nvm I found it in the version.env file